examples/nivisa_dmm_measurement/NIVisaDmmMeasurement.measui

```diff
@@ -1,31 +1,31 @@
 				<Label Height="[float]16" Id="c24c5007263c45aaab35919b80ade906" LabelOwner="[UIModel]3425530122134004b914e5caafe7fa5c" Left="[float]15" Text="[string]Range" Top="[float]16" Width="[float]33" xmlns="http://www.ni.com/PanelCommon" />
-				<ChannelNumericText AdaptsToType="[bool]True" BaseName="[string]Numeric" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/resolution_digits" Height="[float]24" Id="f00e4a6e2da64780a841bd86ebbe1932" IsLabelBoundToChannel="[bool]False" Label="[UIModel]6110b26a8afe4a14b3e3be3bee9d797e" Left="[float]16" TabIndex="[int]1" Top="[float]93" UnitAnnotation="[string]" ValueFormatter="[string]LV:G5" ValueType="[Type]Double" Width="[float]70" />
+				<ChannelNumericText AdaptsToType="[bool]True" BaseName="[string]Numeric" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/resolution_digits" Enabled="[bool]True" Height="[float]24" Id="f00e4a6e2da64780a841bd86ebbe1932" IsLabelBoundToChannel="[bool]False" Label="[UIModel]6110b26a8afe4a14b3e3be3bee9d797e" Left="[float]16" TabIndex="[int]1" Top="[float]93" UnitAnnotation="[string]" ValueFormatter="[string]LV:G5" ValueType="[Type]Double" Width="[float]70" />
 				<Label Height="[float]16" Id="6110b26a8afe4a14b3e3be3bee9d797e" LabelOwner="[UIModel]f00e4a6e2da64780a841bd86ebbe1932" Left="[float]16" Text="[string]Resolution (digits)" Top="[float]73" Width="[float]95" xmlns="http://www.ni.com/PanelCommon" />
 			</ScreenSurfaceCanvas>
 			<Label Height="[float]16" Id="8a81dfd7bc5c46228026d1c21aa98288" LabelOwner="[UIModel]299e88201b64436290271ab48c3f88c1" Left="[float]23" Text="[string]Configuration" Top="[float]120" Width="[float]73" xmlns="http://www.ni.com/PanelCommon" />
-			<ChannelNumericText AdaptsToType="[bool]True" BaseName="[string]Numeric" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Output/measured_value" Height="[float]51" Id="b8c14ae283ec4bf8ad8a037705e08afc" IsLabelBoundToChannel="[bool]False" IsReadOnly="[bool]True" Label="[UIModel]71a08f4917d143caa1a9edb08cfff3c8" Left="[float]195" MinHeight="[float]51" TabIndex="[int]0" Top="[float]30" UnitAnnotation="[string]" ValueFormatter="[string]DisplayFormat=Automatic:Digits=6:DigitDisplayType=DigitsOfPrecision:MinimumFieldWidth=0:AlwaysShowSign=False:ShowThousandsSeparator=False" ValueType="[Type]Double" Width="[float]185">
+			<ChannelNumericText AdaptsToType="[bool]True" BaseName="[string]Numeric" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Output/measured_value" Height="[float]51" Id="b8c14ae283ec4bf8ad8a037705e08afc" IsLabelBoundToChannel="[bool]False" IsReadOnly="[bool]True" Label="[UIModel]71a08f4917d143caa1a9edb08cfff3c8" Left="[float]195" MinHeight="[float]51" TabIndex="[int]0" Top="[float]30" UnitAnnotation="[string]" ValueFormatter="[string]DisplayFormat=Automatic:Digits=6:DigitDisplayType=SignificantDigits:MinimumFieldWidth=0:AlwaysShowSign=False:ShowThousandsSeparator=False" ValueType="[Type]Double" Width="[float]215">
```

Comment:
Does this look the same in InstrumentStudio? When I filed the bug, I experimented with the display format settings and saw some weird output like "9.9e+13 Y" (yottavolt?)

Reply:
![image](https://github.com/ni/measurementlink-python/assets/71399316/dc53d303-094c-45cb-8fe1-2fa861e6d356)
Looks fine in IS to me.